src/Domain/Event/EventHistoryInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Domain\Event;
+
+interface EventHistoryInterface
+{
+    /**
+     * @return \Ramsey\Uuid\UuidInterface[]|[]
```

Comment:
Is this correct? This could be an array of UuidInterface or an array? Also, you could specify the return type.